### ISSUE TITLE
Mobile navigation: tap or swipe

### ISSUE THREAD

**Issue by reginaldofisica** [sozi-projects/Sozi#705]
### Summary of your problem or feature request

In previous versions (at least 19.04) I could navigate just by TAPPING the touch screen on mobile devices. Now I need to SWIPE left or right for the same purpose. Is it a different approach or unexpected behavior?
Thanks for this great software.
Greetings.

### Version of Sozi and other relevant software

* Sozi 23.07.25 
* Drawing by Inkscape 1.2
* (Windows 11)
* Behavior seen playing at Chrome Android 14 - UI 6.1

### Steps to reproduce the problem

Playing any Sozi (23.07) presentation on Smartphones

**Comment by RiccardoRuffilli** [comment]
Is it any way to return to the old behaviour?

**Comment by aumouvantsillage** [comment]
@reginaldofisica 

> Is it a different approach or unexpected behavior?

The short answer is "it is a different approach".

Touch gestures are a contribution from @tshoppa. In the previous version, only mouse actions were supported, "taps" being interpreted as "left clicks". When touch support was added, the "tap as click" behavior ceased to work. And for some reason, @thoppa only implemented swipe gestures.

I have just added tap support as an additional touch gesture. Swipes still work.
